ArduCopter/telemetry.cpp

```diff
@@ -0,0 +1,24 @@
+#include "Copter.h"
+
+//IBus Telemetry (i.e. for fs-i6ab)
+void Copter::init_ibus_telem(){
+    ibus_telem.init();
+    //no need to do things if IBusTelemetry is not activated
+    if(ibus_telem.isInitialized()) {
+        //sensors can easily be added if needed
+        ibus_telem.addSensor(IBUS_MEAS_TYPE_EXTV);
+        ibus_telem.addSensor(IBUS_MEAS_TYPE_ARMED);
+        ibus_telem.addSensor(IBUS_MEAS_TYPE_FLIGHT_MODE);
+    }
+}
+
+void Copter::set_ibus_telem_data(){
+    //no need to do things if IBusTelemetry is not activated
+    if(ibus_telem.isInitialized()) {
+        //sensors can easily be added if needed
+        uint16_t voltage = (uint16_t)(AP_BattMonitor::get_singleton()->voltage()*100);
+        ibus_telem.setSensorMeasurement(1, (int)voltage);
```

Comment:
The sensor lib can grab these values via the singletons removing the need for this new CPP file